### ISSUE TITLE
Invalid query does not return error, but spins forever

### ISSUE THREAD

**Issue by mnalis** [tyrasd/overpass-turbo#667]
When entering the query at wizard at http://overpass-turbo.eu/ which is not recognized, previously error was returned.

Now, it seems that it simply runs "running line" spinner forever, even when just `build query` is pressed (initially I was pressing `build and run query`  and wondering why my seemingly simple/small query is taking so long to complete)

From example, entering in wizard `"bicycle:repair" in Croatia` and clicking `build query` just runs the spinner (I've waited for several minutes and given up so I do not know if it timeouts eventually):
![Screenshot 2023-10-23 at 17-18-01 overpass turbo](https://github.com/tyrasd/overpass-turbo/assets/156656/7b93c698-cf00-405e-bbc1-8ccb43dc3ddd)

Modifying that query to `"bicycle:repair"=* in Croatia` returns correctly in about a second for `build and run query` (and immediately for just `build query`)

Started happening in last few weeks (I guess in or about the same major update which also broke _"save/load from OSM"_ functionality from https://github.com/tyrasd/overpass-turbo/issues/665)

Possibly related issue: https://github.com/tyrasd/overpass-turbo/issues/657#issuecomment-1732616090

**Comment by hfs** [comment]
Overpass Turbo tries to load this file: https://overpass-turbo.eu/assets/en-US-a78e4a15.js
but it’s empty:

```javascript
const e={"en-US":{}};export{e as default};
```

When trying to build a query in the wizard for the first time this exception is logged:

```
Uncaught (in promise) Error: failed to load preset translations file: en-US
    f https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:274
    Xl https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:274
    Gm https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:281
    update_ffs_query https://overpass-turbo.eu/assets/index-9bf03a2e.js:763
    onFfsRun https://overpass-turbo.eu/assets/index-9bf03a2e.js:763
    onFfsBuild https://overpass-turbo.eu/assets/index-9bf03a2e.js:763
    dispatch https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    handle https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    add https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    Bi https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    each https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    each https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    Bi https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    on https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    initClickHandler https://overpass-turbo.eu/assets/index-9bf03a2e.js:763
    each https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    each https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    initClickHandler https://overpass-turbo.eu/assets/index-9bf03a2e.js:763
    Qe https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    In https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    setTimeout handler*Deferred/then/J/< https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    N https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    fireWith https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    fire https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    N https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    fireWith https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    ready https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    setTimeout handler* https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    <anonymous> https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    <anonymous> https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:10
    <anonymous> https://overpass-turbo.eu/assets/urlParameters-314e2e15.js:13
```

When hitting the build query button a second time, the exception mentioned by @mnalis above is logged.

To me it seems there are two problems:

1. The en-US .js file doesn’t seem to contain the content it is supposed to have?
2. The wizard should be able to deal with this error and move on, or display a helpful error message, but not stay in the loading state forever.

**Comment by hfs** [comment]
The preset translation files come from https://github.com/openstreetmap/id-tagging-schema/tree/main/dist/translations

To me it seems that the language-country variants are meant to be merged with the language-without-country.files.

E.g. en.json contains the English presets. en-AU.json only seems to contain the terms that differ from en.json. en-US.json is empty.

I think when the requested language is en-AU, one would need to load en.json, then load en-AU.json and merged it. Entries from en-AU should overwrite entries from en.

What should happen if a user requests a language without translated presets, such as `ee`? It seems best to use English as fall back. So load `en`, then the language, then the language-country if it exists and merge them on top of each other.

**Comment by mnalis** [comment]
> The wizard should be able to deal with this error and move on, or display a helpful error message, but not stay in the loading state forever.

Would just putting that language loading block in `try...catch` fix at least that part? 

It is really making wizard quite annoying to use  (i.e. I've by now had to train myself to never click  on `Build and execute` but to always click first on `Build query` and only if that worked then move over the other side of screen to click `Run`.

**Comment by hfs** [comment]
As a workaround one can set the language to `en` in the settings.

**Comment by mnalis** [comment]
> As a workaround one can set the language to `en` in the settings.

Umm, does that work for you @hfs ? 

I've set language to `auto` (which uses English), but I've now set it to `en` manually, and still trying to build query like `As a workaround one can set the language to en in the settings.` spins in infinity for me.

**Comment by hfs** [comment]
@mnalis, yes, I’ve had the infinite spinning problem for a long time and switching to `en` has fixed it for me. I think I had to reload (F5) one time, but since then it works.

This is what I get now for the invalid query:

![grafik](https://github.com/tyrasd/overpass-turbo/assets/538109/b4c93b13-4bdb-478e-a180-8b64afe4ea95)

**Comment by mnalis** [comment]
@hfs thanks, that workaround works! I've had to (after changing to `en`) press `ctrl-F5`  to force refresh, though.

Still it would be good if it was fixed so it works out of the box for all users. 

(also, I'll link related issue regarding Wizard UX not reporting errors:  in case of  https://github.com/tyrasd/overpass-turbo/issues/403 Wizard however does not spin forever, but instead silently ignores part of input as if it was never entered - and this workaround doesn't work)
